frontend/src/components/Table/ViewPropertyCell.tsx

```diff
@@ -13,6 +13,7 @@ const ViewPropertyCell = (props: CellProps<IProperty, number>) => {
   const property = props.row.original;
   return (
     <Link
+      target="_blank"
```

Comment:
I think you also want: rel="noopener noreferrer"

Reply:
security related tags that help prevent browser hijacking.

Reply:
Ahh ok will add!